cinema/admin.py

```diff
@@ -1,8 +1,23 @@
 from django.contrib import admin
 
-from cinema.models import Movie
+from cinema.models import Movie, Genre, Actor, CinemaHall
 
 
 @admin.register(Movie)
 class MovieAdmin(admin.ModelAdmin):
     pass
+
+
+@admin.register(Genre)
+class GenreAdmin(admin.ModelAdmin):
+    pass
+
+
+@admin.register(Actor)
+class ActorAdmin(admin.ModelAdmin):
+    pass
+
```

Comment:
you can use `admin.site.register(ModelName)` and it would do the same, but without `pass` station